ros2service/ros2service/verb/call.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2016-2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import importlib
+import json
+import time
+
+import rclpy
+from ros2service.api import ServiceNameCompleter
+from ros2service.verb import VerbExtension
+
+
+class CallVerb(VerbExtension):
+    """Call a service."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument(
+            'service_name',
+            help="Name of the ROS service to call to (e.g. '/add_two_ints')")
+        arg.completer = ServiceNameCompleter(
+            include_hidden_services_key='include_hidden_services')
+        parser.add_argument(
+            'service_type',
+            help="Type of the ROS service (e.g. 'std_srvs/Empty')")
+        parser.add_argument(
+            'values', nargs='?', default='{}',
+            help='Values to fill the service request with in JSON format ' +
+                 '(e.g. {"a": 1, "b": 2}), ' +
+                 'otherwise the service request will be published with default values')
+
+        def float_or_int(input_string):
+            try:
+                value = float(input_string)
+            except ValueError:
+                raise argparse.ArgumentTypeError("'%s' is not an integer or float" % input_string)
+            return value
+
+        parser.add_argument(
+            '-l', '--loop', metavar='N', default=None, type=float_or_int,
+            help='Repeat the call every N seconds')
+
+    def main(self, *, args):
+        return main(args)
```

Comment:
I would suggest to call `requester(...)` here and get rid of the `main` function.